package.json

```diff
@@ -7,12 +7,13 @@
   "license": "Apache-2.0",
   "scripts": {
     "clean:deep": "git clean -fqdx .",
-    "build:website": "turbo run build --filter=website",
+    "build:website": "turbo run build:website",
     "lint": "turbo run lint",
-    "typecheck": "turbo run typecheck",
+    "typecheck": "tsc -b",
```

Comment:
should this have `--noEmit`

Reply:
The Polaris example has it emit not sure why.